### ISSUE TITLE
Replication Done with Server Admin instead of Member

### ISSUE THREAD

**Issue by 509dave16** [pouchdb/pouchdb#7252]
### Issue
Replication to the remote CouchDB database is done by the Server Admin(**509dave16**) instead of by the Member(**davidfall**) of the database whose credentials are provided in PouchDB client. In the following reproduction steps, it is shown not only by the Response from the Replication but also by a Create DB request in Postman that an Auth Session for the Server Admin is being used for the replication.
1. Meaning that we can't properly validate in CouchDB because the Server Admin did it instead of the User whose credentials were provided in the PouchDB client.
2. Someone can take the AuthSession Cookie in the Client and start having their way with the CouchDB instance. Like creating a DB, etc.... Even though they were just a simple member.

### Info
- Environment: Browser
- Platform: Chrome
- Adapter: IndexedDB
- Server: CouchDB

### Reproduce
1. Server Admin created in CouchDB and logged in
[![couchdb-server-admin.png](https://s20.postimg.cc/60zheddjx/couchdb-server-admin.png)](https://postimg.cc/image/bp5s59hw9/)
1. User created in CouchDB
[![couchdb-user.png](https://s20.postimg.cc/7fw05knml/couchdb-user.png)](https://postimg.cc/image/o3ni82ie1/)
2. DB created in CouchDB(i.e. todos)
[![couchdb-todo-db.png](https://s20.postimg.cc/n1dbpx4u5/couchdb-todo-db.png)](https://postimg.cc/image/evv9rrgl5/)
3. Make created User a Member of the created DB
[![couchdb-member.png](https://s20.postimg.cc/e6chf0ii5/couchdb-member.png)](https://postimg.cc/image/7snebrdm1/)
4. Add following design doc to throw an error response to show what User requested/performed the Replication
```
{
  "_id": "_design/auth",
  "validate_doc_update": "function (newDoc, oldDoc, userContext, secObject) {\n            var messagesStr = 'context=';\n            for(var key in userContext) {\n                messagesStr += ' | ' + key + ' : ' + userContext[key];\n            }\n            messagesStr += ' security=';\n            for(var key in secObject) {\n                messagesStr += ' | ' + key + ' : ' + secObject[key];\n            }\n            throw({ forbidden: messagesStr});\n        }"
}
```
5. PouchDB client(service, function, etc..) that replicates to CouchDB using the User credentials of the Member
[![pouchdb-service.png](https://s20.postimg.cc/6qd7t6ki5/pouchdb-service.png)](https://postimg.cc/image/85eshwll5/)
6. Make a PUT request with the PouchDB client that will result in a POST request like the following being made to CouchDB.
Request
[![pouchdb-put-request.png](https://s20.postimg.cc/6dltn0x3h/pouchdb-put-request.png)](https://postimg.cc/image/cejik3jpl/)
Response
[![post-response-from-couchdb.png](https://s20.postimg.cc/3wa2fqam5/post-response-from-couchdb.png)](https://postimg.cc/image/ogewe7qd5/)
7. Take the AuthSession Cookie that was seen in the Google Chrome Inspector and place it into a Create DB request in Postman(or some other rest client, etc...)
[![create-db-with-auth-session.png](https://s20.postimg.cc/rai1rn5e5/create-db-with-auth-session.png)](https://postimg.cc/image/cejik1tzd/)
8. You now have a CouchDB database created with an AuthSession Cookie by the Server Admin
[![newly-created-db-from-auth-session.png](https://s20.postimg.cc/p5xoqle25/newly-created-db-from-auth-session.png)](https://postimg.cc/image/bp0q7q3qh/)

**Comment by daleharvey** [comment]
Yup thats how browsers work, nothing really to do with pouchdb. It doesnt manually send the cookies they just get sent along with all other browser requests

**Comment by 509dave16** [comment]
@daleharvey Really appreciate the explanation. I was just puzzled by it. I'm not particularly knowledgeable when it comes to Networking stuff.  What you said makes sense though and that's how I'll be handling things moving forward. Thank you so much!
